### DISCUSSION TITLE
Checkboxes are white

### DISCUSSION THREAD

**Discussion by Shinigami92** [community/community#23587] [Accessibility]
<p>It would be cool if checkboxes will render in a dark(er)<br>
color<br>
If going to notifications, the multiple white checkboxes flashes into your eyes</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/a/2a330572d6c2277dfb39a470d42f0590afe1e5b5.png" data-download-href="/uploads/short-url/61jmIgAxOTqdx3a5776hoPxAg5L.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7195563/181078057-fc837bab-9ad3-4d27-bbd0-66360d766ef6.png" alt="image" data-base62-sha1="61jmIgAxOTqdx3a5776hoPxAg5L" width="113" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/a/2a330572d6c2277dfb39a470d42f0590afe1e5b5_2_113x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/a/2a330572d6c2277dfb39a470d42f0590afe1e5b5_2_169x750.png 1.5x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/a/2a330572d6c2277dfb39a470d42f0590afe1e5b5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">212×938 18.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by 1480c1** [comment]
<p>I would also like to mention the checkboxes that show up when you preview a pr/issue in that the one with an actual check is barely distinguishable from one that has no checkmark<br>
<img src="https://user-images.githubusercontent.com/8345542/181078061-99f3c3ee-1ef9-4398-b4f8-8009942acb16.png" alt="image" data-base62-sha1="w8OqMTBVpMO5BaGEk9CUYEwErot" width="33" height="93"></p>

**Comment by gaknoll** [comment]
<p>Thanks <a class="mention" href="/shinigami92">@Shinigami92</a> and <a class="mention" href="/1480c1">@1480c1</a> for the feedback! We were able to get checkboxes working with light and dark modes in Safari but still need to continue for Chrome. I’ve shared this with the team and made sure it’s in our backlog to look into and hopefully get resolved soon!</p>

**Comment by alrz** [comment, marked as answer]
<p>In chrome, you need to enable <a>#form-controls-dark-mode</a> flag. This fixes the scrollbar and checkbox rendering.</p>

**Reply by rvsfara** [reply]
and does it work in firefox?

**Comment by richiksc** [comment]
<p>Try opening an issue with large checklists (e.g. <a href="https://github.com/microsoft/vscode/issues/111406" rel="noopener nofollow ugc">https://github.com/microsoft/vscode/issues/111406</a>) in Dark Mode. The disabled checkboxes (I’m not an editor) don’t have enough contrast between the checked and unchecked states so it’s really hard to tell what’s checked and what’s not from a quick glance. They also don’t stand out much against the dark background compared to the checkboxes in light mode.</p>
<p>Diff:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/a/aa00d1d1a0c3ef00edd1b1866aed6ae221da5495.jpeg" data-download-href="/uploads/short-url/ofUVpr098yyHXeqsnUjgfuECrR3.jpeg?dl=1" title="lightmode" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/8939680/181078063-c81c17c2-318c-400e-b265-b31207bea643.jpeg" alt="lightmode" data-base62-sha1="ofUVpr098yyHXeqsnUjgfuECrR3" width="690" height="391" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aa00d1d1a0c3ef00edd1b1866aed6ae221da5495_2_690x391.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aa00d1d1a0c3ef00edd1b1866aed6ae221da5495_2_1035x586.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/a/aa00d1d1a0c3ef00edd1b1866aed6ae221da5495.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aa00d1d1a0c3ef00edd1b1866aed6ae221da5495_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">lightmode</span><span class="informations">1264×718 121 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by perkele1989** [comment]
<p>Fully agree!  Looks like they forgot this detail.</p>

**Comment by gaknoll** [comment]
<p>Thanks for sharing this feedback <a class="mention" href="/richiksc">@richiksc</a>! We haven’t forgotten about the checkboxes - they are bothering us as well :wink:. Getting them fixed is in our backlog. I’m going to merge this with a similar topic where <a class="mention" href="/alrz">@alrz</a> has shared a workaround for them in Chrome</p>

**Comment by M0315G** [comment]
<p>While using the Dark mode it’s hard to distinguish between the filled and empty Checkboxes. Maybe adding a little gradient to it will help. The ticks can have a different colour fill.</p>

**Comment by Lachee** [comment]
<p>Its particularly bad with disabled checkboxes.<br>
<img src="https://user-images.githubusercontent.com/1355848/181078068-9e734f29-2bd0-4a41-ace4-28c86ede3d55.png" alt="" data-base62-sha1="zu9q9kLTX2h9hJzL2e2U8euQbKC" width="506" height="194"></p>
<p>I would suggest just making the ticks black or the boxes black, there needs to be some contrast. Gradients wouldn’t really be that appropriate here considering the rest of the style.</p>

**Comment by ArekPiekarz** [comment]
<p>This issue is also affecting me, using Deluminate extension gives much better recognizable state of checkboxes:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/c/9c4fe310bf5c33c4ae4bc3bfecd3bee6fdefdf78.png" data-download-href="/uploads/short-url/miNDPybkaz20UhYe0IaQqSfkl0k.png?dl=1" title="Screenshot_20210103_170128" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3698606/181078070-81a58b44-3acf-4b5a-8b78-68af9c8e01a4.png" alt="Screenshot_20210103_170128" data-base62-sha1="miNDPybkaz20UhYe0IaQqSfkl0k" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9c4fe310bf5c33c4ae4bc3bfecd3bee6fdefdf78_2_690x345.png, https://aws1.discourse-cdn.com/github/original/3X/9/c/9c4fe310bf5c33c4ae4bc3bfecd3bee6fdefdf78.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/c/9c4fe310bf5c33c4ae4bc3bfecd3bee6fdefdf78.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9c4fe310bf5c33c4ae4bc3bfecd3bee6fdefdf78_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20210103_170128</span><span class="informations">732×366 39.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by spencerpogo** [comment]
<p>Any update on this? Using a chrome flag is not an option for me on managed ChromeOS.</p>

**Comment by quyleanh** [comment]
<p>This is my case. Difficult to see checked checkbox. Hope that bug will be fixed soon.</p>
<p><img src="https://user-images.githubusercontent.com/9365035/181078076-e2fabcee-b113-4112-b256-780c82d6b22d.png" alt="image" data-base62-sha1="qGssNKGZLYTpaB11nEKZNdDC771" width="223" height="257"></p>

**Comment by longfellowone** [comment]
<p>Also having this issue, checkbox’s are extremely hard to read with the dark theme</p>

**Comment by cehfisher** [comment]
Hello @Shinigami92 - Thank you for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again! (cc @queenofcorgis)

**Comment by cehfisher** [comment]
Hello @Shinigami92 - the accessibility team reviewed this issue and are happy to report that the notifications checkbox backgrounds all render dark in this scenario. There are other checkbox-related issues they are still working on, but this one is stale. Thank you!
